### PR TITLE
Add psycopg2

### LINKED ISSUE #63: error with psycopg2, connecting to postgresql database
Hi, I tried to follow the instructions to install ODM2PythonAPI via the conda package but I received an error. psycopg2 for connecting to a postgresql database can be a pain to install not sure about including it in a conda package, I don't have experience with them. 

```python
import psycopg2
ImportError: No module named psycopg2
```

### PR DESCRIPTION
**Pull request by ocefpaf**
Should fix https://github.com/ODM2/ODM2PythonAPI/issues/63

@miguelcleon I am assuing you tested the package on Windows in https://github.com/ODM2/ODM2PythonAPI/issues/63#issue-151709897. If so this PR should fix it.

PS: If you open the issue here I will be able to see and probably fix it faster.

### CODE REVIEW

**Comment by miguelcleon** [comment]
Yes, I tested the package on Windows.

**Comment by ocefpaf** [comment]
Thanks @miguelcleon. As soon as we have a green tick here I'll merge and the binaries will be uploaded to the channel.

**Comment by emiliom** [comment]
Looks like we're green now!

So, the end result will be a psycopg2 package on the ODM2 conda channel, or elsewhere? And an updated odm2api conda package that uses the right psycopg2 from the right source?

**Comment by ocefpaf** [comment]
> So, the end result will be a psycopg2 package on the ODM2 conda channel, or elsewhere?

Merges on this repository will upload to the ODM2 channel only.

> And an updated odm2api conda package that uses the right psycopg2 from the right source?

Yep. (But I am not done yet. There is one more change need here.)

File: odm2api/meta.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ requirements:
         - geoalchemy-odm2
         - shapely
         - pandas
-        - psycopg2  # [not win]
+        - psycopg2
```
Comment (ocefpaf) [inline odm2api/meta.yaml:29]: @emiliom I updated the recipe by removing the constraint but I did not bump the build number. That won't trigger the upload. I am fixing that now.

**Comment by emiliom** [comment]
Thanks for the explanations. It just helps me learn more about this complicated workflow thing works.

**Comment by ocefpaf** [comment]
I made that one on purpose for the presentation tomorrow :wink:

**Comment by miguelcleon** [comment]
Should I be able to try this now? Maybe I'm not doing the right thing but I looked at the conda package page and the build number appears the same
from here https://anaconda.org/odm2/odm2api 

odm2api=0.5.0.a0  This is the build number right?

**Comment by ocefpaf** [comment]
You need to wait for the CI to re-build and upload.

You can follow the progress here:

https://ci.appveyor.com/project/ocefpaf/conda-recipes-odm2/branch/master

You can see that some of the new binaries are already there:

https://anaconda.org/odm2/odm2api/0.5.0.a0/download/win-32/odm2api-0.5.0.a0-py27_1.tar.bz2

**Comment by miguelcleon** [comment]
Ok great, thank you!

**Comment by emiliom** [comment]
Looks like the rebuilding and uploading are done now, right? I doubt @miguelcleon will test it tonight (it's pretty late on the East coast), but I just wanted to confirm it's officially ready. Thanks!

**Comment by ocefpaf** [comment]
FYI the Windows builds are always the slowest. AppVeyor does not build the matrix concurrently and waits a few minutes between the builds. There is very little we can do about that but to wait.

**Comment by emiliom** [comment]
Thanks, @ocefpaf. I wasn't in any hurry myself. I guess what I was ultimately asking is, what do we look for (and where) to know when it's completely done and ready?

**Comment by ocefpaf** [comment]
> Thanks, @ocefpaf. I wasn't in any hurry myself.

No problem. I think that understanding the nature of each CI is important.

> I guess what I was ultimately asking is, what do we look for (and where) to know when it's completely done and ready?

Ideally the the [channel](http://anaconda.org/ODM2) like @miguelcleon did. However, since that PR was just merged and you can follow the CI build by clicking on the badges [here](https://github.com/ODM2/conda-recipes-ODM2/).

**Pull request merged**